src/pool/srv_target.c

```diff
@@ -1744,9 +1750,10 @@ ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 			D_FREE(arg);
 		}
 	} else {
-		D_WARN("Ignore update pool "DF_UUID" %d -> %d\n",
-		       DP_UUID(pool->sp_uuid), pool->sp_map_version,
-		       map_version);
+		/* This should be a D_DEBUG eventually. */
+		D_INFO(DF_UUID ": ignored pool map update: version=%u->%u cached_version=%u\n",
+		       DP_UUID(pool->sp_uuid), pool_map_get_version(pool->sp_map), map_version,
+		       pool->sp_map_version);
```

Comment:
Just curious, keep this to INFO is because you want to "avoid" the duplicate pool map update rpc?

Reply:
It will not be easy to avoid the duplicate map update RPC from happening, I think, for the PS leader would need to track which engines have what map version, and `iv` would need to export some interface to support partial `SYNC`s---too much complexity. I'm keeping this at INFO level because I suspect people may still rely on this message for debugging purposes. Perhaps when things become more stable we should change this INFO to DEBUG.